### DISCUSSION TITLE
Feature request: Make it possible to change technical contact on an organization level

### DISCUSSION THREAD

**Discussion by hjaltedaniel** [umbraco/Umbraco.Cloud.Issues#817] [Ideas]
### Issue description

When you have an organization on Umbraco Cloud, it should be possible to set a technical contact for all projects under the organization instead of having to do it individually for every project.

**Comment by FransdeJong** [comment]
To add to this issue:

I would like to have a default technical contacts for the organization that gets added on creating a new project instead of the creator.
I would also like the same with the Account contacts.

**Reply by Zhaph** [reply]
Yes please! At the moment, people can create a project in an org, and no-one else has access until they are invited in, which feels like a dangerous oversight.

**Comment by sajumb** [comment]
Thanks for this great feature request. We have created a ticket in our product backlog and marked it as an candidate for upcoming sprints.

**Comment by bjarnef** [comment]
Relates to this:
https://github.com/umbraco/Umbraco.Cloud.Issues/discussions/810

It would be great to change default technical contact. Often we as developers create Umbraco Cloud projects, but we mainly want a project manager (which technical skills) to be technical contact and/or a tech lead.

Settings the default on organization level would make sense.
